spec/index.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Spec proposal</title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class="remove"></script>
+    <script class='remove'>
+      // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+      var respecConfig = {
+        specStatus: "CG-DRAFT", // FIXME: not a CG-DRAFT, ED?
+        shortName: "lws-ucs",
+        copyrightStart:  "2024",
+        // previousPublishDate:  "N/A",
+        editors: [
+          { name: "Pierre-Antoine Champin" },
+          { name: "Hadrian Zbarcea" },
+        ],
+        group: "solid",  // FIXME: https://respec.org/w3c/groups/ does not list 'lws' yet
+        wg: "Linked Web Storage",
+        maxTocLevel: 4,
+      };
+    </script>
+  </head>
+  <body>
+    <section id="abstract">
+      <p>
+        User stories and use cases for the LWS spec.
+      </p>
+    </section>
+    <section id="sotd">
+      <p>
+        This is an unofficial proposal.
+      </p>
+    </section>
+
+    <section id="introduction">
+      <h2>Introduction</h2>
+      <p>
+        See <a href="https://github.com/w3c/respec/wiki/User's-Guide">ReSpec's user guide</a>
+        for how toget started!
+      </p>
+    </section>
+
+    <section id="use-cases">
+      <h2>Use Cases</h2>
+      <p>The use cases for this specification are documented below.</p>
+      <div data-include="user-stories.md" data-format="markdown"></div>
+    </section>
+    <section id="glossary">
+      <h2>Glossary</h2>
+      <p>The glossary provides definitions of key terms and concepts used throughout this specification. It serves as a quick reference to ensure clarity and a shared understanding among readers and contributors. By standardizing terminology, the glossary supports consistent communication and alignment in interpreting this specification.</p>
+      <div data-include="glossary.md" data-format="markdown"></div>
```

Comment:
```suggestion
      <div data-include="glossary.md" data-include-format="markdown" data-include-replace="true"></div>
```

* `data-format` was a mistake (although for some reason, it kinda worked)
* `data-include-replace` gets rid of the enclusing `div`, which prevented section numbering to work